hugo/assets/scss/components/section.scss

```diff
@@ -62,6 +62,12 @@
         }
     }
 
+    &--blog {
```

Comment:
we can just use the regular section title, ignore the design here (old section title styling was used there)